src/napari_spatialdata/utils/_utils.py

```diff
@@ -279,3 +281,29 @@ def _adjust_channels_order(element: SpatialImage | MultiscaleSpatialImage) -> tu
         new_raster = list_of_xdata
 
     return new_raster, rgb
+
+
+def _get_sdata_key(sdata: EventedList, elements: dict[str, dict[str, str | int]], key: str) -> tuple[SpatialData, bool]:
+    """
+    Get the index of SpatialData object and key of SpatialElement.
+
+    Parameters
+    ----------
+    sdata: EventedList
+        EventedList containing the SpatialData objects currently associated with the viewer.
+    elements
+    key: str
+        The name of the item in the element widget.
+
+    Returns
+    -------
+    tuple[int, str]
```

Comment:
the return is of a different type

Reply:
Resolved with [020c28a](https://github.com/scverse/napari-spatialdata/pull/136/commits/020c28a3da3a91463c4a10ca42d3fc6d4307cbdb)